apps/agreement/test/agreement/agreement_new_action.js

```diff
@@ -154,17 +150,23 @@ contract('Agreement', ([_, owner, submitter, someone]) => {
                     })
                   }
 
-                  context('when the transaction fees module is set', () => {
-                    context('when the transaction fee is zero', () => {
+                  context('when the app fees cashier module is set', () => {
+                    let aragonAppFeesCashier
+
+                    beforeEach('load app fees cashier', async () => {
```

Comment:
Theoretically this doesn't need to be a `beforeEach()`, right?

Reply:
We do, some of these tests are testing different settings with the arbitrator, therefore we cannot trust it is not going to change